lib/active_model/serializer/adapter/json_api/relationship.rb

```diff
@@ -0,0 +1,53 @@
+module ActiveModel
+  class Serializer
+    module Adapter
+      class JsonApi
+        class Relationship
```

Comment:
Agreed. Under JsonApi is fine, see my errors pr

On Mon, Feb 22, 2016 at 9:56 PM Ben Mills <notifications@github.com> wrote:

> In lib/active_model/serializer/adapter/json_api/relationship.rb
> <https://github.com/rails-api/active_model_serializers/pull/1529#discussion_r53733792>
> :
>
> > @@ -0,0 +1,53 @@
> > +module ActiveModel
> > +  class Serializer
> > +    module Adapter
> > +      class JsonApi
> > +        class Relationship
>
> namespacing it under ApiObjects seems unnecessary to me.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rails-api/active_model_serializers/pull/1529/files#r53733792>
> .
>